### PR TITLE
CB-8357 - added global ROLE_LIMITED_CLUSTER_ADMIN for CM users

### PR DESCRIPTION
**Pull request by attilapalfi92**
See detailed description in the commit message.

### CODE REVIEW

**Comment by attilapalfi92** [comment]
datalake creation in progress 4 trying this out

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerLdapService.java [hunk 1]
```diff
@@ -36,6 +36,8 @@
 public class ClouderaManagerLdapService {
     private static final Logger LOGGER = LoggerFactory.getLogger(ClouderaManagerLdapService.class);
 
+    private static final String ROLE_LIMITED_CLUSTER_ADMIN = "ROLE_LIMITED_CLUSTER_ADMIN";
```
Comment (keyki) [inline cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerLdapService.java:39]: Should be an env var as the others.

**Comment by keyki** [comment]
According to the ticket we need a dashboard user as well.

**Comment by harshy-cloudera** [comment]
cluster admin role addition looks good.

**Pull request merged**